apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -436,7 +436,11 @@ def _write_dataframe(
         soma_df = factory.open(df_uri, "w", soma_type=DataFrame)
     except DoesNotExistError:
         soma_df = DataFrame.create(
-            df_uri, schema=arrow_table.schema, platform_config=platform_config
+            df_uri,
+            schema=arrow_table.schema,
+            platform_config=platform_config,
+            ### XXX TEMP
```

Comment:
Something to note?

Reply:
what was I saying here? maybe "did something get missed here?" instead

Reply:
temp/debug; deleted; thank you